cadence/tests/flow_evm_bridge_tests.cdc

```diff
@@ -497,5 +507,13 @@ fun bridgeNFTFromEVM(signer: Test.TestAccount, contractAddr: Address, contractNa
         signer
     )
     Test.expect(bridgeResult, Test.beSucceeded())
+
+    var events = Test.eventsOfType(Type<NonFungibleToken.Withdrawn>())
+    let withdrawnEvent = events[events.length - 1] as! NonFungibleToken.Withdrawn
+    Test.assertEqual(bridgeAccount.address, withdrawnEvent.from!)
```

Comment:
I would like to add some assertions to check the `withdrawnEvent.id` & `depositedEvent.id` event fields, but I am not sure how to convert from the `erc721ID` argument to a Cadence NFT ID. Any ideas @sisyphusSmiling

Reply:
I think the best way to do this would be to look for the subsequent `IFlowEVMNFTBridge.BridgedNFFromEVM` event and match the `evmID` value to `erc721`. This event will also contain the `id` value corresponding to the bridged `nft.id` value which can then be checked in the `Withdrawn` and `Deposited` events. This is a bit roundabout, but necessary given the fact `evmID` is not guaranteed to match `nft.id` due to different types - `UInt256` and `UInt64` respectively.